src/makeCalculator.js

```diff
@@ -4,7 +4,26 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(operation, num) {
+      this.result = operation(this.result, num);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
```

Comment:
The `add`, `subtract`, `multiply`, and `divide` functions are defined as standalone functions. To align with the design of the calculator object, these should be methods that operate on the `result` property. Consider modifying them to use `this.result` and update the result accordingly.